plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -171,6 +174,9 @@ func (p *Plugin) Admit(ctx context.Context, a admission.Attributes, o admission.
 	case resourceSliceResource:
 		return p.admitResourceSlice(nodeName, a)
 
+	case csrResource:
+		return p.admitCSR(nodeName, a)
```

Comment:
discussed in sig-auth today, put this behind a deprecated opt-out feature gate to give folks piggy-backing on kubelet credentials doing weird things with CSRs a 1-2 release out if this disrupts them